prosemirror-jdita/tsconfig.json

```diff
@@ -63,7 +63,7 @@
     /* Experimental Options */
     "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-
+    "skipLibCheck": true,
```

Comment:
Why is this change in this PR? Seems inappropriate!

Reply:
This was necessary for the documentation generation to work, `typedoc` was trying to pull imports from 3rd party libs which some of them had issues, that resulted in generation failure.
eg: 
```shell
$ typedoc --treatWarningsAsErrors --out generated-docs src/index.ts
../node_modules/prosemirror-commands/dist/index.d.ts:2:25 - error TS7016: Could not find a declaration file for module 'prosemirror-state'. 'C:/Users/z/Documents/work/prosemirror-jdita/node_modules/prosemirror-state/dist/index.js' implicitly has an 'any' type.
  If the 'prosemirror-state' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module 'prosemirror-state';`

2 import { Command } from 'prosemirror-state';
                          ~~~~~~~~~~~~~~~~~~~
````

`skipLibCheck` tells `typescript` to ignore anything outside of current project and not worry about it.

Reply:
Hmm I don't think that is entirely correct. I just read the documentation here - https://www.typescriptlang.org/tsconfig#skipLibCheck

Reply:
Yes it effects `typescript` compilation of the code, By enabling it we won't try to check any files in `node_modules`.

Reply:
@adamretter The reason we need this in documentation generation task and not in building the project even though both of them run `tsc` is: `typedoc` tried to explore more files, while regular `tsc` is bounded by `tsconfig.json` 
```json
"rootDir": "src" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

We can limit `typedoc` reach using `skipLibCheck`.

Reply:
@marmoure I still don't think that `skipLibCheck` is the correct solution here. Can you look for alternative options for configuring the paths that tsc investigates please?